x-pack/plugins/secops/public/pages/network/network.tsx

```diff
@@ -56,7 +56,9 @@ const NetworkComponent = pure<NetworkComponentProps>(({ filterQuery }) => (
                       sourceId="default"
                       startDate={from}
                     >
-                      {({ kpiNetwork }) => <KpiNetworkComponent data={kpiNetwork} />}
+                      {({ kpiNetwork, loading }) => (
+                        <KpiNetworkComponent data={kpiNetwork} loading={loading} />
```

Comment:
In testing your branch, I noticed that subsequent datetime updates would not show the loading spinner. This is fixable by updating the `KpiNetworkComponent` to be managed ala the `NetworkTopNFlowTable` a little below.

To create the managed component, just add the following above the `NetworkComponent` declaration:

``` javascript 
const KpiNetworkComponentManage = manageQuery(KpiNetworkComponent);

```

Then update the reference to `KpiNetworkComponent` to be:

``` javascript
{({ kpiNetwork, loading, id, refetch }) => (
  <KpiNetworkComponentManage
    id={id}
    setQuery={setQuery}
    refetch={refetch}
    data={kpiNetwork}
    loading={loading}
  />
)}
```

And lastly, you'll need to add the `refetch` prop to `public/containers/kpi_network/index.tsx` just as you added `loading` in this PR, but that should do it. Let me know if you have any questions -- so happy to see the loading spinner in non-table components now :)

Reply:
Thanks for pointing this out @spong! I didn't event aware about the use case!